src/coreclr/src/System.Private.CoreLib/src/System/Attribute.CoreCLR.cs

```diff
@@ -542,11 +361,6 @@ public static Attribute[] GetCustomAttributes(ParameterInfo element, Type attrib
             if (element.Member == null)
                 throw new ArgumentException(SR.Argument_InvalidParameterInfo, nameof(element));
 
-
-            MemberInfo member = element.Member;
```

Comment:
Is this going to change behavior when the arguments passed into this method are 3rd party non-runtime reflection objects?

Reply:
It's working around the `RuntimeParameterInfo` being a bit broken (not following the inheritance chain if `m_tkParamDef` is a null Token, but not considering that it may be inherited); so reimplements the inheritance search here

Reply:
Also not passing inherit to the CustomAttribute methods for these types probably didn't help